### ISSUE TITLE
Sync Gamedata Says No Drives 

### ISSUE THREAD

**Issue by Dark98** [CryoByte33/steam-deck-utilities#149]
My SD Card isn't mounted/formatted by steam so the mount point is /mnt & not the below

https://github.com/CryoByte33/steam-deck-utilities/blob/341e69ea7c4de9a1bca407fbfbef1540cbd1386b/internal/config.go#L131

**Comment by CryoByte33** [comment]
Did you format your drive manually? The KDE formatter also uses the directory currently in the code (by default), so I assumed it would be safe to assume it was there.

I would be fine with adding a second directory, but /var isn't where drives should be mounted in general (https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/var.html) so I'd rather not add anything in there to the list.

**Comment by Dark98** [comment]
> Did you format your drive manually? The KDE formatter also uses the directory currently in the code (by default), so I assumed it would be safe to assume it was there.
> 
> I would be fine with adding a second directory, but /var isn't where drives should be mounted in general (https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/var.html) so I'd rather not add anything in there to the list.

Yeah I manually formatted as I have several partitions on my SD Card (For Windows, Etc)

When I check '/mnt' on the Deck it shows the mount folder & in the fstab it's mounted to '/mnt' & not '/var/mnt' but SteamOS does some weird stuff & when I used the path '/mnt' it wouldn't work but it works with '/var/mnt'
